src/aria2.cpp

```diff
@@ -127,9 +154,6 @@ Aria2::Aria2():
 
 void Aria2::close()
 {
-  MethodCall methodCall("aria2.pauseAll", m_secret);
-  doRequest(methodCall);
-
```

Comment:
I had the same question, and I chose to keep this change because it contributes to a more gradual way of explaining the problem and leading to (and justifying) the most robust solution.